src/Aspire.Hosting.Testing/DistributedApplicationFactory.cs

```diff
@@ -243,6 +243,13 @@ private void EnsureEntryPointStarted()
                 {
                     EnsureDepsFile(_entryPoint);
 
+                    _appHostProjectPath = ResolveProjectPath(_entryPoint.Assembly);
+                    if (!string.IsNullOrEmpty(_appHostProjectPath))
+                    {
+                        // FIXME: reset to old one at some point?
+                        Environment.CurrentDirectory = _appHostProjectPath;
```

Comment:
I don't think we should be changing the global CWD during these tests. That would cause problems if you are running multiple tests in parallel.

I think the original problem is in WithRealmImport.

https://github.com/dotnet/aspire/blob/024b48bf581f6790b0878128c6acff7863d3d5f7/src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs#L141-L146

That Directory.Exists check isn't right. The `importDirectory` is relative to the apphost directory. See the next level down in `.WithBindMount`:

https://github.com/dotnet/aspire/blob/024b48bf581f6790b0878128c6acff7863d3d5f7/src/Aspire.Hosting/ContainerResourceBuilderExtensions.cs#L78-L80

I think we should just remove that Directory.Exists call to fix the original issue.

Reply:
Great feedback, thanks!

Reply:
Done.